### ISSUE TITLE
mdc-select default browser styling - Edge

### ISSUE THREAD

**Issue by kvdheuvel** [material-components/material-components-web#4088]
### What MDC Web Version are you using?
0.41.0

### What browser(s) is this bug affecting?
We got an issue in Edge, but the styling is set for all browsers.

### What OS are you using?
Windows 10

### What are the steps to reproduce the bug?
Open the mdc-select in Edge and select an value. Reopen the select and you will see that the selected value can't be seen because the background color and text color are white.

I've created an codepen based on the provided template:
[https://codepen.io/anon/pen/NEjeYw](https://codepen.io/anon/pen/NEjeYw)

### What is the expected behavior?
The selected option should be readable.

### What is the actual behavior?
Option background-color & color are the same color.

### Any other information you believe would be useful?
/

**Comment by abhiomkar** [comment]
Thanks for reporting this issue @kvdheuvel!

Seems like a bug on Edge where it doesn't allow CSS variables on native select > option element. Here is a quick [example on jsfiddle](http://jsfiddle.net/sL3boua4/16) that demonstrates this bug.

I just sent a PR #4089 to fix it.

**Comment by kvdheuvel** [comment]
Hello @abhiomkar,

I don't think that the CSS variables are the problem. I think the problem is that the selected option has the same color as `--mdc-theme-surface` (both white). If you change the --mdc-theme-surface color to let's say green you can see (read) the selected option because that text is white.

**Comment by abhiomkar** [comment]
You're right. If I set the background color to something else it changes the color of option element on edge except when I set it to hard coded value instead of using CSS variable. See this [example](http://jsfiddle.net/sL3boua4/25). I find this weird. Browser might be preventing it from doing this for accessibility reasons.

Also I'm not able to change the text color of `option` element on edge, it only lets me change the background color.

I'm inclined towards removing styles from native `option` element altogether.
